admin/app/adapters/user.js

```diff
@@ -18,4 +18,12 @@ export default class UserAdapter extends ApplicationAdapter {
   urlForUpdateRecord(id) {
     return `${this.host}/${this.namespace}/admin/users/${id}`;
   }
+
+  updateRecord(store, type, snapshot) {
+    if (snapshot.adapterOptions && snapshot.adapterOptions.anonymizeUser) {
+      const url = this.urlForUpdateRecord(snapshot.id) + '/anonymize';
+      return this.ajax(url, 'POST');
+    }
+    return super.updateRecord(...arguments);
+  }
```

Comment:
Dans le cas où seule l'URL est modifiée, moi je recommande plutôt de faire la modif dans la surcharge de `urlForUpdateRecord()`. Surcharger `updateRecord()` c'est plutôt utile quand on souhaite modifier le payload ou ce genre de choses.
```suggestion
  urlForUpdateRecord(id, modelName, { adapterOptions }) {
    const url = `${this.host}/${this.namespace}/admin/users/${id}`;

    if (adapterOptions && adapterOptions.anonymizeUser) {
      delete adapterOptions.anonymizeUser;
      return url + '/anonymize';
    }

    return url;
  }
```

Reply:
Merci Laura, seulement ici on est dans un cas où on change aussi la méthode en POST.

Reply:
Pas vu !! Pardon les gars